build.include

```diff
@@ -17,6 +17,7 @@ THEIA_BRANCH="master"
 THEIA_COMMIT_SHA=
 THEIA_GIT_REFS="refs\\/heads\\/master"
 THEIA_DOCKER_IMAGE_VERSION=
+ARCH=
```

Comment:
should this have a default value, of the current `uname -m` value? 

or...

```
ARCH="$(uname -m)" && \
if [[ ${ARCH} == "x86_64" ]]; then \
  export ARCH="amd64"; \
elif [[ ${ARCH} == "aarch64" ]]; then \
  export ARCH="arm64"; \
fi
```

Reply:
or, if we want to FORCE users to set the --arch flag, then should we fail the build if --arch is not passed in?

Reply:
The default value is not needed as the build.sh will be called by each arch in their respective builds, which will set the ARCH value.

We need not force users to pass --arch flag. Have introduced the --arch flag for Travis only to differentiate between the images built on different archs. If users don't specify --arch flag, the normal flow will execute as was done in GA.

Reply:
if the flag is just here to label the image differently, it should not be introduced.
`TAG` argument should be enough then.